src/Stryker.Core/Stryker.Core/Baseline/Utils/BaselineMutantHelper.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+using Stryker.Core.Mutants;
+using Stryker.Core.Reporters.Json;
+
+namespace Stryker.Core.Baseline.Utils
+{
+    public class BaselineMutantHelper : IBaselineMutantHelper
+    {
+        public IEnumerable<Mutant> GetMutantMatchingSourceCode(IEnumerable<Mutant> mutants, JsonMutant baselineMutant, string baselineMutantSourceCode)
+        {
+            return mutants.Where(x =>
+               x.Mutation.OriginalNode.ToString() == baselineMutantSourceCode &&
+               x.Mutation.DisplayName == baselineMutant.MutatorName);
+        }
+
+        public string GetMutantSourceCode(string source, JsonMutant baselineMutant)
```

Comment:
I wonder why this method is needed? We can always ask the original and mutated sourcecode for mutants. Why can we not do that for JsonMutants? Maybe it's better to include a reference to the original mutant in the JsonMutant and then you can ask the sourceCode text there?

Reply:
Like we discussed. We need this method to get the source code because we are unable to get a reference to the original mutant. This method is used when we load in an excisting report from stryker-dashboard. Maybe this can be refined when we implement the line and span info from libgit